### DISCUSSION TITLE
Defining Diffusion Coefficient as a function of SOC

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#4542] [Q&A] [answered]
I have a relation between diffusion coefficient of electrodes wrt SOC. Can we provide it as an input in pybamm model?

**Comment by rtimms** [comment, marked as answer]
Yes, diffusivity can be a function of stoichiometry and temperature. You need to define a function as part of your parameter set, e.g. like this: https://github.com/pybamm-team/PyBaMM/blob/develop/src/pybamm/input/parameters/lithium_ion/Ecker2015.py#L5
